enterprise/internal/own/ownref_test.go

```diff
@@ -290,3 +305,50 @@ func TestBagUnverifiedEmailOnlyMatchesWithItself(t *testing.T) {
 		})
 	}
 }
+
+func initUser(ctx context.Context, t *testing.T, db edb.EnterpriseDB) (*types.User, error) {
```

Comment:
`t.Helper()` will help more precise error reporting

```suggestion
func initUser(ctx context.Context, t *testing.T, db edb.EnterpriseDB) (*types.User, error) {
  t.Helper()
```

Reply:
yes, forgot to add, adding now